packages/devtools_app/lib/src/inspector/flutter/inspector_tree_flutter.dart

```diff
@@ -423,7 +424,7 @@ class _InspectorTreeState extends State<InspectorTree>
 }
 
 final _defaultPaint = Paint()
-  // TODO(kenz): try to use color from Theme.of(context) for treeGuidelineColor
+// TODO(kenz): try to use color from Theme.of(context) for treeGuidelineColor
```

Comment:
I assume this is the output of running dartfmt.

Reply:
Right!